FirebaseAuth/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Unreleased
+- [fixed] Allow http schemes when using the auth emulator. (#7350)
```

Comment:
Nit: This describes what was done but doesn't get into what it enables, maybe something like "Auth emulators now work across the local network"?  I'm fine landing as is but was curious if you like the suggestion.

Reply:
Done.